analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -187,7 +189,7 @@ abstract class PackageManager(
      * Return a tree of resolved dependencies (not necessarily declared dependencies, in case conflicts were resolved)
      * for all [definitionFiles] which were found by searching the [analyzerRoot] directory.
      */
-    open fun resolveDependencies(analyzerRoot: File, definitionFiles: List<File>): ResolutionResult {
+    open fun resolveDependencies(definitionFiles: List<File>): ResolutionResult {
```

Comment:
I'd be nice to assert (in the first line of this function) that the definition file is actually underneath the analyzer root - but needs not necessarily be part of this PR.

Reply:
Right, I'll leave this as a future exercise, as such an assertion could have already been present before, but it isn't.